library/src/main/java/net/rafaeltoledo/okir/CounterManager.java

```diff
@@ -0,0 +1,17 @@
+package net.rafaeltoledo.okir;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+class CounterManager {
+    private static AtomicInteger counter;
```

Comment:
I don't know if the counter should be static. If the instance is reused between tests, this can be a problem.